mysql-test/suite/rpl/t/rpl_gtid_deployment_step.test

```diff
@@ -73,11 +73,6 @@ select @@global.gtid_executed;
 -- connection master
 call master_procedure();
 select @@global.gtid_executed;
--- connection slave
--- disable_warnings
--- source include/start_slave.inc
```

Comment:
This looks suspicious, we basically remove starting slave here which is a significant change in the behavior.
Consider adding
`--let $slave_io_errno= convert_error(XXX)`
or
`--let $slave_sql_errno= convert_error(XXX)`
before `-- source include/start_slave.inc` instead.

Reply:
In this case slave is already running and master was restarted. `start_slave.inc` can return Last_IO_Errno=2003 that is `error reconnecting to master 'root@127.0.0.1:13001' - retry-time: 1  retries: 6`.
To solve this issue `start_slave.inc` should be replaced with `sync_slave_sql_with_master.inc` that reconnects slave with master, but `sync_slave_sql_with_master.inc` is already used a few lines later.